extensions/toggl-unofficial/README.md

```diff
@@ -0,0 +1,15 @@
+# Toggl Unofficial
+
+This is a small extension I've built for [Raycast](https://raycast.com), designed to allow easy starting/stopping of Toggl timers. 
+
+I'll be submitting this extension to the Raycast Store shortly, so check back in a day or two for the published version.
+
+## Actions
+- Start Toggl Timer
+    - automatically retrieves unique time entries from the last 30 days
+- Stop Toggl Timer (not completed)
```

Comment:
What about this, is this the one you added?

Reply:
Yup, fixed up some other things in the README too. Sorry about that.